release/ansible/Dockerfile.rhel8

```diff
@@ -22,8 +22,11 @@ ENV HOME=/opt/ansible \
     USER_UID=1001
 
 # get artifacts from cachito
-COPY $REMOTE_SOURCES/cachito-gomod-with-deps/app $REMOTE_SOURCES_DIR
-WORKDIR $REMOTE_SOURCES_DIR/app
+COPY $REMOTE_SOURCES $REMOTE_SOURCES_DIR
+
+# TODO: remove this line (debugging)
```

Comment:
Did we still want this debugging line?

Reply:
Just got out of the ansible call. I think we want this for now just to verify things are working as expected

Reply:
Yea, I want it for this import so I can see what is going on. I'll remove it in another PR once we know things are working.